apps/console/src/features/identity-providers/components/settings/authenticator-settings.tsx

```diff
@@ -682,19 +682,19 @@ export const AuthenticatorSettings: FunctionComponent<IdentityProviderSettingsPr
                     } else if (prop.key === "commonAuthQueryParams") {
                         prop.displayName = "Additional query parameters";
                         prop.description = "These  will be sent to the identity provider as query parameters in the " +
-                            "authentication request. \nE.g., loginHint=hint1"
+                            "authentication request. \nE.g., loginHint=hint1";
+                    } else if (prop.key === "IsBasicAuthEnabled") {
+                        prop.description = "Specify whether to enable HTTP basic authentication for authenticating " +
```

Comment:
Can we think of improving this description?

So the use of this option is to decide whether we send client_id and client_secret as request parameters or in the basic auth header.

Reply:
The description has been updated.